assets/src/application/ui/forms/modalForm/RenderModalForm.tsx

```diff
@@ -58,17 +44,18 @@ const FormModalForm: React.FC<FormModalFormProps> = ({
 			title={title}
 			visible={true}
 			onOk={onClose}
-			style={modalStyle}
 			okText={__('Submit')}
 			okButtonProps={submitButton}
 			cancelText={__('Reset')}
 			cancelButtonProps={resetButton}
 			onCancel={onClose}
 			width={'80%'}
+			wrapClassName='ee-modal-form'
+			bodyStyle={{ padding: 0 }}
```

Comment:
hmm, is this a temp?

Reply:
No, it's to avoid hardcoded padding set by AntD, I have added that padding in `.scss`